pkg/executor/client/client.go

```diff
@@ -201,3 +201,36 @@ func (c *Client) _tapService(ctx context.Context, tapSvcReqs []TapServiceRequest
 	}
 	return nil
 }
+
+// IsValid will return true if the service uid and address are found
+func (c *Client) IsValid(ctx context.Context, fn *fv1.FunctionWithAddress) (string, error) {
+	executorURL := c.executorURL + "/v2/isValid"
+
+	body, err := json.Marshal(fn)
+	if err != nil {
+		return "", errors.Wrap(err, "could not marshal request body for checking function with address")
+	}
+
+	req, err := retryablehttp.NewRequestWithContext(ctx, "POST", executorURL, bytes.NewReader(body))
+	if err != nil {
+		return "", errors.Wrap(err, "could not create request for checking function with address")
+	}
+	req.Header.Set("Content-Type", "application/json")
+
+	resp, err := c.httpClient.Do(req)
+	if err != nil {
+		return "", errors.Wrap(err, "error posting to checking function with address")
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode != 200 {
+		return "", ferror.MakeErrorFromHTTP(resp)
+	}
+
+	svcName, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return "", errors.Wrap(err, "error reading response body from checking function with address")
+	}
+
+	return string(svcName), nil
```

Comment:
The return type should be a boolean value

Reply:
I agree. Changed the return type. Please check it out